package.json

```diff
@@ -9,6 +9,7 @@
         "tooling/noir_js_types",
         "tooling/noirc_abi_wasm",
         "tooling/noir_js",
+        "tooling/noir_js_backend_barretenberg",
```

Comment:
I merged in 2902 which is why the diff is quite a lot in this PR

Reply:
Changed the base from master to a branch which acccumulates the changes in this PR, so this is no longer relevant